keyboards/fjlabs/bolsa65/keymaps/via/keymap.c

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2020 <me@homedrop.org>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _LAYER0,
+    _LAYER1,
+    _LAYER3,
+    _LAYER4,
```

Comment:
VIA keymap doesn't compile without this change.

```suggestion
    _LAYER2,
    _LAYER3,
```

Reply:
Interesting, must have inroduced this typo sometime after my compile.